Addons.ini

```diff
@@ -147,3 +147,10 @@ DownloadUrl64=https://github.com/cot6/reshade-addons/releases/download/setup-rel
 PackageName=RenoDX by ShortFuse
 PackageDescription=A toolset to mod games, short for "Renovation Engine for DirectX Games". Currently it can replace shaders, inject buffers, add overlays, upgrade swapchains, upgrade texture resources ...
 RepositoryUrl=https://github.com/clshortfuse/renodx
+
+[22]
+PackageName=3DGameBridge by Janthony & DinnerBram
+PackageDescription=Enable stereoscopic 3D on LeiaSR displays for SBS content
+DownloadUrl32=https://github.com/JoeyAnthony/3DGameBridgeProjects/releases/latest/srReshade.zip
+DownloadUrl64=https://github.com/JoeyAnthony/3DGameBridgeProjects/releases/latest/srReshade.zip
```

Comment:
These are missing a `/download` in the URL at the end I believe, so should be: https://github.com/JoeyAnthony/3DGameBridgeProjects/releases/latest/download/srReshade.zip
Otherwise it don't seem to point somewhere valid, with the fixed URL one is currently redirected to https://github.com/JoeyAnthony/3DGameBridgeProjects/releases/download/v0.3.0/srReshade.zip (which doesn't exist because v0.3.0 doesn't have srReShade.zip yet, but the next v1.0.0 has, so once that release is published, it would automatically redirect to v1.0.0 and start working).

Reply:
Good point, I updated it in commit: c25dcea0d7e9303011848209671ac5abf427f481